CODE_OF_CONDUCT.md

```diff
@@ -72,7 +72,7 @@ You may report in the following ways:
 In any situation, you may send an email to <support@adafruit.com>.
 
 On the Adafruit Discord, you may send an open message from any channel
-to all Community Helpers by tagging @community helpers. You may also send an
+to all Community Helpers by tagging @community moderators. You may also send an
```

Comment:
Can you update the cookiecutter as well please? (If it hasn't been updated yet.) Thanks!

Reply:
Yeah, it has. I think I did that a few weeks ago.